salesagility/SuiteCRM
# Email client compose listview/detailview

## Description



- Added the new email composer to listview, detailview and bulk email action

## How To Test This

- Rebuild everything.
- Check that email composer opens in listview/detail view when clicking on an email.
- Check that the fields are auto-populated with the correct information for that module.

## Types of changes


### Final checklist
